### DISCUSSION TITLE
Add font size tween for TMP_Text

### DISCUSSION THREAD

**Discussion by SametHope** [KyryloKuzyk/PrimeTween#129] [Ideas]
`Tween.TextMaxVisibleCharacters` exists so I think `Tween.TextFontSize` would be a nice QoL to have as well. 

Notice this would be different than just tweening the scale of the texts transform due to alignment, wrapping, overflow etc features of the text. I also know it is possible to use `Tween.Custom` to achieve this but that could be said for lots of existing features as well.

**Comment by KyryloKuzyk** [comment]
I added the `TextFontSize()` in 1.3.0 :)
https://github.com/KyryloKuzyk/PrimeTween/blob/main/changelog.md
